rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,180 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.3
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka (bukka@php.net), Larry Garfield (larry@garfieldtech.com)
+  * Status: Draft
+
+===== Introduction =====
+
+The Open Source project that develops the PHP language and its reference implementation uses an RFC process to discuss
+and vote on proposed changes. This was created mainly for user facing changes. It works well for that purpose but it is
+not that effective for deciding about purely technical changes that impact PHP internals and extension API's. It is not
+clear how it should resolve conflicts on the technical basis between contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src when there is a disagreements between core developers.
```

Comment:
```suggestion
This RFC proposes an introduction of the PHP Technical Committee (TC) that would mediate code contribution conflicts between core developers proposing or committing changes to php-src.
```

"Decide about all technical aspects" is vague and will be argued back and forth. The goal is to mediate conflict which arise due to changes to php-src.

Reply:
The technical aspects are actual defined so it should be clear what it is and if it is not, we should update the definition. The current statement is more vague IMO as it basically covers RFC proposed changes which is explicitly limited later in the RFC. It means that TC should never decide about commits that introduce user visible changes unless there are technical issues with that.